AppBuilder/platform/views/viewComponent/ABViewGridComponent.js

```diff
@@ -254,9 +254,11 @@ export default class ABViewGridComponent extends ABViewComponent {
                   this.onAfterSelect(data, preserve);
                }
             },
-            onBeforeEditStart: function (id) {
-               if (!this.getItem(id) == "appbuilder_select_item") return false;
-            },
+            // onBeforeEditStart: function (/*id*/) {
+            //    // Not sure what this is suposed to check, but this condition
+            //    // will always be false.
+            //    if (!this.getItem(id) == "appbuilder_select_item") return false;
+            // },
```

Comment:
OK, this is odd.  I happened to see this in my GitHub client.  It shows this modification as: 
```
-            onBeforeEditStart: function (id) {
-               if (this.getItem(id) !== "appbuilder_select_item") return false;
-            },
+            // onBeforeEditStart: function (/*id*/) {
+            //    // Not sure what this is suposed to check, but this condition
+            //    // will always be false.
+            //    if (!this.getItem(id) == "appbuilder_select_item") return false;
+            // },
```

The ` if (this.getItem(id) !== "appbuilder_select_item")` seems like a valid check to make sure that the value in `this.getItem(id)` isn't `appbuilder_select_item`.

Reply:
I did try that, but I believe it broke editing some things in the Grid. My suspicion is that this is left over from when we were using `selectivity`?